### ISSUE TITLE
Reactions are not accessible with Voiceover

### ISSUE THREAD

**Issue by heull001** [signalapp/Signal-iOS#5497]
Reactions on Messages are not accessible with Voiceover. The Lockscreen-Notification is accessible, but in the App there is no ability to get access.

**Comment by heull001** [comment]
This problem is not solved yet.

**Comment by cannona** [comment]
Can confirm. It is not possible to hear the reactions on messages when using VoiceOver.

**Comment by cannona** [comment]
This issue should remain open, as it does not appear to have been resolved
in the latest production version.

**Comment by Miguel-Signal** [comment]
@cannona @heull001 Could you please update to the [latest version of Signal](https://apps.apple.com/us/app/signal-private-messenger/id874139669) and see if this resolves the issue? Then, if the issue is still happening, could you write in to support@signal.org with your debug logs from this new version?

To share a debug log from Signal iOS, tap your profile Profile Avatar, Settings, Help, Submit Debug Log, Copy Link, then open this email, long-press and choose paste to paste the link.

**Comment by DGrothe-PhD** [comment]
Can confirm the issue on ios 16.6.1

**Comment by marissa-signal** [comment]
Hi - This will be fixed in app version 6.54, which will go to beta next week.
